app/(afterLogin)/(payroll)/employee-information/[id]/page.tsx

```
@@ -158,13 +158,8 @@ const EmployeeProfile = () => {
         bordered={false}
       >
         <Row gutter={[32, 32]}>
-          <Col sm={24} md={24} xs={24} lg={10} xl={10}>
-            <Card
-              loading={isLoading}
-              className={`mb-3 ${isMobile ? 'w-full m-0' : ''}`}
-              style={isMobile ? { width: '100%' } : {}}
-              bordered={!isMobile}
-            >
+          <Col lg={8} md={10} xs={24}>
+            <Card loading={isLoading} className={`mb-3 ${isMobile ? 'w-full m-0' : ''}`} style={isMobile ? { width: '100%' } : {}} bordered={!isMobile}>
```

ESLint

[error] Replace `·loading={isLoading}·className={`mb-3·${isMobile·?·'w-full·m-0'·:·''}`}·style={isMobile·?·{·width:·'100%'·}·:·{}}·bordered={!isMobile}` with `⏎··············loading={isLoading}⏎··············className={`mb-3·${isMobile·?·'w-full·m-0'·:·''}`}⏎··············style={isMobile·?·{·width:·'100%'·}·:·{}}⏎··············bordered={!isMobile}⏎············`

(prettier/prettier)